packages/synapse-react-client/src/components/CertificationQuiz/CertificationQuiz.tsx

```diff
@@ -191,16 +192,32 @@ const CertificationQuiz: React.FunctionComponent = () => {
             <Button
               onClick={() => window.open(GETTING_STARTED_URL, '_blank')}
               className="help-button"
-              color="secondary"
+              color="primary"
               variant="contained"
             >
-              <HelpOutlineTwoTone
-                className="HelpButton"
-                style={{ marginRight: '4px', color: 'white' }}
-              />
-              Help
+              <Box sx={{ display: 'flex', alignItems: 'center' }}>
+                <HelpOutlineTwoTone
+                  className="HelpButton"
+                  style={{ marginRight: '4px', color: 'white' }}
+                />
+                <>Help</>
+              </Box>
             </Button>
-            <div dangerouslySetInnerHTML={{ __html: quiz.header }}></div>
```

Comment:
We need to style this section, but I'm sure this is controversial.  
We could modify the quiz header to not contain the quiz name or html, and just contain a plain text version of the description.  I suggest we implement the next time the quiz description needs to change.  By that time, the UI will be pointing at this snapshot in history, and we can modify the quiz value without impacting users.  Then we can switch this code to pull in the quiz.header again

Reply:
As long as we remember to make that change, sounds good to me!